docs/workflows/create_sync_publish.rst

```diff
@@ -112,6 +112,34 @@ Assuming your token is ``YourRepositoryToken``, create the remote with the ``sle
     policy='on_demand' \
     sles_auth_token='YourRepositoryToken'
 
+.. _create-uln-remote:
+
+Creating an ULN remote
+**********************
+
+ULN stands for "Unbreakable Linux Network" and refers to the way Oracle in particular, `delivers Oracle Linux repositories <https://linux.oracle.com/>`_ to their enterprise customers.
+If you have access, you can use an ULN remote to synchronize repositories from an ULN server to Pulp.
```

Comment:
```suggestion
You can use an ULN remote to synchronize repositories from an ULN server to Pulp.
```
I decided the "If you have access" contains no real useful information (it is essentially a tautology and also redundant with the next sentence that makes clear that you "must provide your ULN login credentials").

Reply:
Done.